### PR TITLE
Facilitate Packit support for Fedora Linux 38 and Fedora Linux 39

### PR DESCRIPTION
**Pull request by gridhead**
Stepped down the version requirement for `python-gitlab`, updated the Packit configuration to allow for making COPR builds, testing package, synchronizing with the downstream, proposing to the downstream, making production builds, making builds on Koji and making updates on Bodhi and added some installability checks

This should help with #122

### CODE REVIEW

File: .fmf/version [hunk 1]
```diff
@@ -0,0 +1 @@
+1
```
Comment (Zlopez) [inline .fmf/version:1]: Why this empty file?

File: .packit.yaml [hunk 2]
```diff
@@ -2,21 +2,88 @@
 # https://packit.dev/docs/configuration/
 
 specfile_path: pagure-exporter.spec
+issue_repository: https://github.com/gridhead/protop2g
```
Comment (Zlopez) [inline .packit.yaml:5]: Shouldn't this be pagure-exporter?

**Review by Zlopez — COMMENTED** [review]
Just few comments, otherwise looks good

**Comment by onuralpszr** [comment]
@gridhead  I don't see f38 builds, you missing those to add it.

File: .packit.yaml (hunk 2 above)
Comment (gridhead) [reply .packit.yaml:5]: Yep.

This should be https://github.com/fedora-infra/pagure-exporter.

**Pull request merged**